test/tools/libtesteth/TestSuite.cpp

```diff
@@ -56,5 +61,20 @@ void TestSute::runAllTestsInFolder(const std::string& _folder) const
 	}
 }
 
+void TestSuite::copyAllTestsFromFolder(string const& _testFolder) const
```

Comment:
Will it be used for any tests besides Transaction tests? If not, maybe leave it in `TransactionTestSuite`